test/uk/gov/hmrc/icedsubscriptionfrontend/services/AuthServiceSpec.scala

```diff
@@ -115,49 +121,54 @@ class AuthServiceSpec extends SpecBase with MockAuthConnector {
 
     "user has no enrolment" must {
       "return NotEnrolled" in {
-        stubAuth() returns Future.successful(emptyEnrolments and Some(User) and Some(AffinityGroup.Organisation) and Some(ggwCreds))
+        stubAuth() returns Future.successful(
+          emptyEnrolments and Some(User) and Some(AffinityGroup.Organisation) and Some(ggwCreds))
 
         service.authenticate().futureValue shouldBe AuthResult.LoggedIn(UserType.NotEnrolled)
       }
     }
 
     "user has a different enrolment" must {
       "return NotEnrolled" in {
-        stubAuth() returns Future.successful(otherEnrolments and Some(User) and Some(AffinityGroup.Organisation) and Some(ggwCreds))
+        stubAuth() returns Future.successful(
+          otherEnrolments and Some(User) and Some(AffinityGroup.Organisation) and Some(ggwCreds))
 
         service.authenticate().futureValue shouldBe AuthResult.LoggedIn(UserType.NotEnrolled)
       }
     }
 
     "user HMRC-SS-ORG enrolment is not active" must {
       "return NotEnrolled" in {
-        stubAuth() returns Future.successful(
-          Enrolments(Set(ssEnrolment.copy(state = "disabled"))) and Some(User) and Some(AffinityGroup.Organisation) and Some(ggwCreds))
+        stubAuth() returns Future.successful(Enrolments(Set(ssEnrolment.copy(state = "disabled"))) and Some(User) and Some(
+          AffinityGroup.Organisation) and Some(ggwCreds))
 
         service.authenticate().futureValue shouldBe AuthResult.LoggedIn(UserType.NotEnrolled)
       }
     }
 
     "user HMRC-SS-ORG enrolment is active" must {
       "return AlreadyEnrolled" in {
```

Comment:
Maybe add a test where there is an active enrolment but no eori is defined and so get `AlreadyEnrolled(None)`. (Otherwise final case in AuthService `getIdentifier(...)` is not covered.)